lib/server.js

```diff
@@ -27,6 +27,16 @@ app.use(notFound);
 app.use(errorHandler);
 
 // TODO: JSDoc Comment
+/**
+ * @param  {} port||process.env.PORT||3000
+ * @param  {} (
+ * @param  {} =>{console.log(`ServerRunningonPort${port||process.env.PORT||3000}`
+ * @param  {} ;}
+ * @param  {true} ;constconfig={useNewUrlParser
+ * @param  {true} useCreateIndex
+ * @param  {true};mongoose.connect(process.env.MONGODB_URI} useUnifiedTopology
+ * @param  {} config
+ */
```

Comment:
Hmm, this looks like maybe it was auto generated but not checked; lots of params here that shouldn't be